lib/Conversion/TosaToTTIR/TosaToTTIRPatterns.cpp

```diff
@@ -162,17 +185,33 @@ void addCompareOpsConversionPatterns(MLIRContext *ctx,
       tosa::GreaterOp, mlir::tt::ttir::GreaterThanOp>>(typeConverter, ctx);
 }
 
+void addClampOpConversionPattern(MLIRContext *ctx, RewritePatternSet &patterns,
+                                 TypeConverter &typeConverter) {
+  patterns.add<TosaToTTIRClampOpConversionPattern>(typeConverter, ctx);
+}
+
+void addMultiplyOpConversionPattern(MLIRContext *ctx,
+                                    RewritePatternSet &patterns,
+                                    TypeConverter &typeConverter) {
+  patterns.add<TosaToTTIRMultiplyOpConversionPattern>(typeConverter, ctx);
+}
+
 } // namespace
 
 namespace mlir::tt {
 
 void populateTosaToTTIRPatterns(MLIRContext *ctx, RewritePatternSet &patterns,
                                 TypeConverter &typeConverter) {
+  // for conversion patterns that directly use the
+  // TosaToTTIRDefaultDPSOpConversionPattern
```

Comment:
All comments must be full sentences, beginning with a capital letter and ending with a full stop.